solidity/test/helpers/FormulaFunctions.js

```diff
@@ -129,15 +129,15 @@ const balancedWeights = (primaryReserveStakedBalance, primaryReserveBalance, sec
     }
     expect(q.gt(0) && p.gt(0)).to.be.true('ERR_INVALID_RESERVE_RATE');
 
-    const tp = t.mul(p);
-    const rq = r.mul(q);
+    const tq = t.mul(q);
```

Comment:
You should change the calls to this method in the test themselves. Otherwise, they will fail

Reply:
I did not find any such call when I made this change. Did you actually see any of the tests failing because of that?